platform/ios/app/Info.plist

```diff
@@ -27,9 +27,11 @@
 	<key>NSHumanReadableCopyright</key>
 	<string>© 2014–2017 Mapbox</string>
 	<key>NSLocationAlwaysUsageDescription</key>
-	<string>The map will ALWAYS display the user’s location.</string>
+	<string>The map will always display your location.</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>The map will display the user’s location.</string>
+	<string>The map will display your location.</string>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>The map will potentially always display your location.</string>
```

Comment:
This wording equivocates about whether the location will be displayed. I thought the intention behind this string was to help the user choose between two options. If that’s the case, would this be too verbose?

> The map will display your location. If you choose Always, the map may also use your location when it isn’t visible in order to improve OpenStreetMap and Mapbox products.

Reply:
> I thought the intention behind this string was to help the user choose between two options.

This is correct, of course. We’ve been ignoring the intention of these keys because this is an internally-facing debug app. (Slight aside: If we were to strictly follow iOS 11 guidance on `always`, we wouldn’t offer it at all — this app has no user feature that benefits from that mode.)

That said, we might as well make these realistic strings and I like the one you’ve come up with. 👍

Reply:
<img width="320" alt="screen shot 2017-08-28 at 12 06 18 pm" src="https://user-images.githubusercontent.com/1198851/29782125-5800602a-8be9-11e7-9be6-e5da72371955.png">